app/views/case_workers/admin/allocations/_allocation.html.haml

```diff
@@ -62,7 +62,7 @@
                 %td
                   %span.js-test-case-number
                     = link_to claim.case_number , case_workers_claim_path(claim)
-                    %div.unique-id-small= claim.unique_id
+                    %span.unique-id-small= claim.unique_id
```

Comment:
@zheileman I fixed it with css but I am to ashamed to actually tag the line where I did it..

Git will give me away anyway...damn you git!